### PR TITLE
Text block legal, link-farm variations automation

### PR DESCRIPTION
**Pull request by sigadamvenkata**
Text block legal, link-farm variations automation
Ran all text block suite (total 8) cases are passing

### CODE REVIEW

File: features/milo/text.block.spec.js [hunk 1]
```diff
@@ -5,37 +5,49 @@ module.exports = {
       tcid: '0',
       name: '@Text',
       path: '/drafts/nala/blocks/text/text',
-      tags: '@text @smoke @regression @milo',
+      tags: '@text @smoke @regression @milo,',
```
Comment (skumar09) [inline features/milo/text.block.spec.js:8]: please remove the `','` after `@milo` tag

File: selectors/milo/text.block.page.js [hunk 2]
```diff
@@ -105,7 +130,7 @@ export class Text {
  *
  * @param {string} textType - The type of the Text block to verify.
  * Possible values are 'text', 'text (Intro)', 'Text (full width)'.
- * text (full-width, large), text (long form, large), and text (inset, large, m spacing)
+ * text (full-width, large), text (long form, large), and text (inset, large, m spacing), text (legal), text (link-farm)
```
Comment (skumar09) [inline selectors/milo/text.block.page.js:133]: very minor comment: please add to new line, ( `max allowed line length is 100 char` )

Comment (skumar09) [reply selectors/milo/text.block.page.js:133]: @sigadamvenkata : PR looks good, just few mionr review comments

**Review by skumar09 — CHANGES_REQUESTED** [review]

**Review by chadsunvice — APPROVED** [review]

File: features/milo/text.block.spec.js (hunk 1 above)
Comment (sigadamvenkata) [reply features/milo/text.block.spec.js]: Done santosh. thanks
https://github.com/adobecom/nala/pull/146/commits/4170bc33f6092d0e4249924df168a5c0333e3670

File: selectors/milo/text.block.page.js (hunk 2 above)
Comment (sigadamvenkata) [reply selectors/milo/text.block.page.js]: done, please check
https://github.com/adobecom/nala/pull/146/commits/5a31d887a733d3ed4bddb41c3579fbe534d87f7d

**Review by skumar09 — APPROVED** [review]

**Pull request merged**